File: python/program/Laptop.py
from Elektronik import Elektronik

# Class Laptop (Child dari Elektronik)
class Laptop(Elektronik):
    def __init__(self, id="", namaProduk="", harga=0,
                 kategori="", jumlahStok=0, statusBarang="",
                 merek="", warna="", processor=""):
        super().__init__(id, namaProduk, harga, kategori, jumlahStok, statusBarang)
        self.merek = merek
        self.warna = warna
        self.processor = processor

    # Getter & Setter
    def getMerek(self):
        return self.merek

    def setMerek(self, merek):
        self.merek = merek

    def getWarna(self):
        return self.warna

    def setWarna(self, warna):
        self.warna = warna

    def getProcessor(self):
        return self.processor

    def setProcessor(self, processor):
        self.processor = processor

File: php/program/Laptop.php
<?php
require_once "Elektronik.php";

class Laptop extends Elektronik {
    private $merek;
    private $warna;
    private $processor;

    // Constructor default
    public function __construct($id = "", $namaProduk = "", $harga = 0,
                                $kategori = "", $jumlahStok = 0, $statusBarang = "",
                                $merek = "", $warna = "", $processor = "", $fotoProduk = "") {
        parent::__construct($id, $namaProduk, $harga, $kategori, $jumlahStok, $statusBarang, $fotoProduk);
        $this->merek = $merek;
        $this->warna = $warna;
        $this->processor = $processor;
    }

    // Getter & Setter
    public function getMerek() {
        return $this->merek;
    }

    public function setMerek($merek) {
        $this->merek = $merek;
    }

    public function getWarna() {
        return $this->warna;
    }

    public function setWarna($warna) {
        $this->warna = $warna;
    }

    public function getProcessor() {
        return $this->processor;
    }

    public function setProcessor($processor) {
        $this->processor = $processor;
    }
}
